### DISCUSSION TITLE
Welcome to LanguageLeapAI Discussions!

### DISCUSSION THREAD

**Discussion by SociallyIneptWeeb** [SociallyIneptWeeb/LanguageLeapAI#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by LW-Trix** [comment]
Hello! I've been having some trouble and issues following the guide, eventually it all works out but I sometimes get either too confused or completely cannot get what I have to do. Will there be a detailed tutorial at some point specifically or only a small update on the guide?

Also the main issue right now is figuring out how I have to update the VOICE_IDs, I completed the curl request but afterwards I cannot comprehend what I have to do to continue. Perhaps the names of the folders are the ID's? But I did check and they were the same before and after I ran the command.

**Reply by LW-Trix** [reply]
Oops, my mistake, I did not realize the newest video is a guide on how to do it, excuse me for not paying attention and thanks for making a guide

**Comment by yoyogift** [comment]
is it can only use in Nvidia? My computer is intel😟

**Comment by dracon57** [comment]
hi so i managed to get stuck on the first step of cloning the repository how may I go about doing that, also I did watch the video but I am a little still not quite sure what I am doing wrong

**Comment by l84d8sw8** [comment]
alright so i did everything, including watching the guide like 4 times, but its just not working. something io found when running the ID command for the mic and all that is that it starts at 14 and onwards. would i translate 14 to = 1? and when i look at docker it  says not connected to hub, would that be something thats not making it work?

**Reply by l84d8sw8** [reply]
something that i forgot to include is that running the ID for python opens a new cmd tab, but only opens for like 1 second then automatically closes

**Comment by iamjackfrostings** [comment]
Thank you very much. I'm totally ignorant to coding but I got the Voice --> Translation --> JP Voice done due to the easy tutorial! 

Is there a way to incorporate Open Ai's Chat GPT to input the translation into the ChatGPT and take the output from it to VoiceVox?
Everytime I try to import openai it has an error with the httpx version. 

I could be totally wrong but from my understanding google translate api uses httpx version 13.3 but openai downloads version 25.1 by default.  As well as something with gradio requiring a specific version of pydantic. Is there any way to bypass this?
